docs/modules/spark-k8s/partials/supported-versions.adoc

```diff
@@ -3,9 +3,9 @@
 // Stackable Platform documentation.
 // Please sort the versions in descending order (newest first)
 
-- 4.0.0 (Hadoop 3.4.1, Scala 2.13, Python 3.11, Java 17) (Experimental)
-- 3.5.6 (Hadoop 3.3.4, Scala 2.12, Python 3.11, Java 17) (LTS)
-- 3.5.5 (Hadoop 3.3.4, Scala 2.12, Python 3.11, Java 17) (Deprecated)
+- 4.0.1 (Hadoop 3.4.1, Scala 2.13, Python 3.11, Java 17) (Experimental)
+- 3.5.6 (Hadoop 3.3.4, Scala 2.12, Python 3.11, Java 17) (Deprecated)
+- 3.5.7 (Hadoop 3.3.4, Scala 2.12, Python 3.11, Java 17) (LTS)
 
 Some reasons why Spark 4 is considered experimental (as of August 2025):
```

Comment:
Iceberg 1.10 added Spark 4 support: https://iceberg.apache.org/releases/#1100-release
Can we please remove the "Missing Iceberg compatibility" line? (I can not comment on this unchanged line)

Reply:
thx!